src/accounts/api/serializers.py

```diff
@@ -81,3 +84,34 @@ def validate(self, data):
                 "Verification token seems invalid or maybe outdated. Try requesting a new one."
             )
         return account
+
+class ChangePasswordSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Account
+        fields = ("password",)
+        extra_kwargs = {"password": {"write_only": True}}
+    
+
+class ChangePasswordRequestSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = PasswordResetRequestModel
+        fields = ["email"]
+
+    email = serializers.EmailField()
+
+    def validate(self, data):
+        email = data.get("email")
+        account = Account.objects.filter(email=email).first()
+        # Note: Make this silently fail later; users don't need to know about this.
+        if account is None:
+            raise serializers.ValidationError("Account with this email doesn't exist.")
+
+        # Create a new instance of PasswordResetRequestModel using the account's verification token
+        token = secrets.token_urlsafe(32)
+        new_model_data = {"token": token, "account": account, "created_at": timezone.now() + timezone.timedelta(minutes=35)}
```

Comment:
```suggestion
        new_model_data = {"token": token, "account": account)}
```
You don't need to pass anything here. The models needs to register the creation date time itself.